docs/networking.md

```diff
@@ -459,20 +459,46 @@ $ kops create cluster \
 
 You can adjust Cilium agent configuration with most options that are available in [cilium-agent command reference](http://cilium.readthedocs.io/en/stable/cmdref/cilium-agent/).
 
-E.g enabling logstash integration would require you to change above block to
+The following command will launch your cluster with desired Cilium configuration
 
+```console
+$ kops update cluster myclustername.mydns.io --yes
 ```
+
+##### Enabling BPF NodePort
+
+As of Kops 1.18 you can safely enable Cilium NodePort.
```

Comment:
I'm skeptical of the value of documenting in which version each feature was added. This should just state what NodePort is and how one can enable it.

Reply:
I thought this would be the most helpful since Kops doesn't have versioned docs. If this goes in, it would reference features that doesn't exists.